shared/src/test/scala/squants/thermal/TemperatureSpec.scala

```diff
@@ -47,32 +52,39 @@ class TemperatureSpec extends FlatSpec with Matchers {
     x.toKelvinScale should be(0)
     x.toFahrenheitScale should be(-459.67)
     x.toCelsiusScale should be(-273.15)
+    x.toScale(Rankine) should be(0)
```

Comment:
toKelvinScale, toCelsiusScale, and toFahrenheitScale are public convenience methods defined on the squants.thermal.Temperature class (see https://github.com/typelevel/squants/blob/master/shared/src/main/scala/squants/thermal/Temperature.scala#L112). The general form is Temperature.toScale(unit: TemperatureScale). 

It makes sense to have these convenience methods for Kelvin, Celsius, and Fahrenheit, since they're so common. I chose not to defined a convenience method for Rankine since it's rarer. But if people feel strongly about it, I can add it.